domains/adamski.json

```diff
@@ -0,0 +1,12 @@
+{
+  "owner": {
+    "username": "edtimer",
+    "email": "b4timer@protonamil.com"
+  },
+  "record": {
+    "CNAME": "adamski.pages.dev"
```

Comment:
Can you add more stuff to your website?

Reply:
Oh, alright noted. Thought I can take my time to design it better.

Reply:
You can add like an about me to your website for example

Reply:
@CuteDog5695  Thank you. I have uploaded more to the site. Regards.